### ISSUE TITLE
表格解析异常

### ISSUE THREAD

**Issue by yangyu92** [flutter/flutter_markdown#294]
表格格式不正确时: 包含无效表头与底部多余空行时就奔溃啦;

源文件如下: 
```
|  |  |
| - | - |
| 最大圆角厚度 | 45mm |
| 最小加工板材尺寸刀具直径 | 45mm |
| 刀具直径 | 150mm |
| 转速 | 3100rpm |
| 进刀速度 | 10～5000MM/min |
| 退刀速度 | 10～5000MM/min |
|   |    |   (该处多余空行)
```

**Comment by domesticmouse** [comment]
I believe the above translates as:

When the table format is incorrect: it crashes when it contains invalid headers and extra blank lines at the bottom;

The source file is as follows:

@mjordan56 I'm guessing this is going to be a set of test cases for `markdown`? =)

**Comment by mjordan56** [comment]
There was a bug in the builder routine that caused an assert to be thrown in this case. The problem was that the last line with empty table cell data was leaving the build routine indicating an inline builder hadn't completed. This has been fixed so empty table cells are properly handled.

Table tests have been added to check for instances of empty table cell data.

**Comment by domesticmouse** [comment]
Fix is in v0.5.0
